README.md

```diff
@@ -43,7 +43,7 @@ Depending on your operating system, you will need to install:
 
 ### On Windows
 
-Install the current version of Python from the [Microsoft Store package](https://docs.python.org/3/using/windows.html#the-microsoft-store-package).
+Install the current version of Python from the [Python Website](https://www.python.org/downloads/windows/).
```

Comment:
Maybe instead of replacing we could list both options?

Reply:
I do not see the advantage of selecting the python.org option but I do see the downside of selecting it.

Reply:
I agree with @richardlau